Karl/schema_karl.h

```diff
@@ -35,35 +35,134 @@ SOFTWARE.
 namespace current {
 namespace karl {
 
+// Karl's startup parameters.
```

Comment:
Looks like this should be `Karl's Nginx parameters.` and `startup parameters` are declared below.